README.md

```diff
@@ -8,6 +8,41 @@ Yet another note taking tool.
 cargo tauri dev
 ```
 
+## Setup environment
+This guide focuses on how to setup your development system based on ubuntu:jammy.
+
+1. Install dependencies
+    ```bash
+    sudo apt-get update
+    sudo apt-get install -y \
+        libwebkit2gtk-4.0-dev \
+        build-essential \
+        curl \
+        wget \
+        file \
+        libssl-dev \
```

Comment:
The same goes here: we do not directly depend on the package, please remove it.

```suggestion
```

Reply:
This package is remove with commit c337707716c614b8271430b3999e47c3e33cc656.